provider/datagen/src/transform/cldr/percent/mod.rs

```diff
@@ -0,0 +1,92 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use std::collections::HashSet;
+
+use crate::provider::IterableDataProviderInternal;
+use crate::transform::cldr::cldr_serde;
+
+use icu_dimension::provider::*;
+use icu_provider::prelude::*;
+use icu_provider::DataProvider;
+use tinystr::tinystr;
+
+impl DataProvider<PercentEssentialsV1Marker> for crate::DatagenProvider {
+    fn load(&self, req: DataRequest) -> Result<DataResponse<PercentEssentialsV1Marker>, DataError> {
+        self.check_req::<PercentEssentialsV1Marker>(req)?;
+        let langid = req.locale.get_langid();
+
+        let numbers_resource: &cldr_serde::numbers::Resource = self
+            .cldr()?
+            .numbers()
+            .read_and_parse(&langid, "numbers.json")?;
+
+        let result = extract_percent_essentials(numbers_resource);
+
+        Ok(DataResponse {
+            metadata: Default::default(),
+            payload: Some(DataPayload::from_owned(result?)),
+        })
+    }
+}
+
+impl IterableDataProviderInternal<PercentEssentialsV1Marker> for crate::DatagenProvider {
+    fn supported_locales_impl(&self) -> Result<HashSet<DataLocale>, DataError> {
+        Ok(self
+            .cldr()?
+            .numbers()
+            .list_langs()?
+            .map(DataLocale::from)
+            .collect())
+    }
+}
+
+fn extract_percent_essentials<'data>(
+    numbers_resource: &cldr_serde::numbers::Resource,
+) -> Result<PercentEssentialsV1<'data>, DataError> {
+    let percent_formats = &&numbers_resource
+        .main
+        .value
+        .numbers
+        .numsys_data
+        .percent_patterns
+        .get(&tinystr!(8, "latn"))
```

Comment:
Note: We'll want to support multiple numbering systems; see how we do it in DecimalSymbols. Please add a TODO.

Reply:
Added some TODOs on top like how it was in Currency. I checked through a few of the different locales in [CLDR data](https://github.com/unicode-org/cldr-json/blob/main/cldr-json/cldr-numbers-full/main/el/numbers.json#L94) and only ever saw the one `percentFormats-numberSystem-latn` property.

Is just the TODO good for now? Or am I looking in the wrong spot for those other numbering systems?

Reply:
TODOs are fine.